test/e2e/apps/job.go

```diff
@@ -72,6 +72,7 @@ var _ = SIGDescribe("Job", func() {
 	f.NamespacePodSecurityLevel = admissionapi.LevelPrivileged
 	parallelism := int32(2)
 	completions := int32(4)
+	ready := int32(2)
```

Comment:
This is used in only one test, and it equals parallelism (by the test intention), so no need to introduce a new param.